musicbot/bot.py

```diff
@@ -1393,6 +1393,26 @@ def _remove_url_from_autoplaylist(self, url):
         write_file(self.config.auto_playlist_file, self.autoplaylist)
         log.debug("Removed {} from autoplaylist".format(url))
 
+    async def handle_timeout(self, guild: discord.Guild):
+        event = self.server_specific_data[guild]["timeout_event"]
+
+        try:
+            log.debug(
```

Comment:
I'd prefer this to be log.info as it's not an bug issue or anything funny going on.